src/components/Section/Relationships/Relatives/Relative.jsx

```diff
@@ -19,65 +17,164 @@ import {
+      label: i18n.m('relationships.relatives.label.frequency.monthly'),
+      value: 'Monthly',
+    }, {
+      name: 'frequency-quarterly',
+      label: i18n.m('relationships.relatives.label.frequency.quarterly'),
+      value: 'Quarterly',
+      className: 'frequency-quarterly',
+    }, {
+      name: 'frequency-annually',
+      label: i18n.m('relationships.relatives.label.frequency.annually'),
+      value: 'Annually',
+    }, {
+      name: 'frequency-other',
+      label: i18n.m('relationships.relatives.label.frequency.other'),
+      value: 'Other',
+    },
+  ])
+
+  getRelativeOptions = () => ([
+    {
+      name: 'relation-mother',
+      label: i18n.m('relationships.relatives.label.relation.mother'),
+      value: 'Mother',
+    }, {
+      name: 'relation-father',
+      label: i18n.m('relationships.relatives.label.relation.father'),
+      value: 'Father',
+    }, {
+      name: 'relation-stepmother',
+      label: i18n.m('relationships.relatives.label.relation.stepmother'),
+      value: 'Stepmother',
+    }, {
+      name: 'relation-stepfather',
+      label: i18n.m('relationships.relatives.label.relation.stepfather'),
+      value: 'Stepfather',
+    }, {
+      name: 'relation-fosterparent',
+      label: i18n.m('relationships.relatives.label.relation.fosterparent'),
+      value: 'Fosterparent',
+    }, {
+      name: 'relation-child',
+      label: i18n.m('relationships.relatives.label.relation.child'),
+      value: 'Child',
+    }, {
+      name: 'relation-stepchild',
+      label: i18n.m('relationships.relatives.label.relation.stepchild'),
+      value: 'Stepchild',
+    }, {
+      name: 'relation-brother',
+      label: i18n.m('relationships.relatives.label.relation.brother'),
+      value: 'Brother',
+    }, {
+      name: 'relation-sister',
+      label: i18n.m('relationships.relatives.label.relation.sister'),
+      value: 'Sister',
+    }, {
+      name: 'relation-stepbrother',
+      label: i18n.m('relationships.relatives.label.relation.stepbrother'),
+      value: 'Stepbrother',
+    }, {
+      name: 'relation-stepsister',
+      label: i18n.m('relationships.relatives.label.relation.stepsister'),
+      value: 'Stepsister',
+    }, {
+      name: 'relation-halfbrother',
+      label: i18n.m('relationships.relatives.label.relation.halfbrother'),
+      value: 'Half-brother',
+    }, {
+      name: 'relation-halfsister',
+      label: i18n.m('relationships.relatives.label.relation.halfsister'),
+      value: 'Half-sister',
+    }, {
+      name: 'relation-fatherinlaw',
+      label: i18n.m('relationships.relatives.label.relation.fatherinlaw'),
+      value: 'Father-in-law',
+    }, {
+      name: 'relation-motherinlaw',
+      label: i18n.m('relationships.relatives.label.relation.motherinlaw'),
+      value: 'Mother-in-law',
+    }, {
+      name: 'relation-guardian',
+      label: i18n.m('relationships.relatives.label.relation.guardian'),
+      value: 'Guardian',
+    },
+  ])
+
```

Comment:
The options defined in these functions seem like they'd be good to store in another file eventually.